docs/user-management/rbac/projects.md

```diff
@@ -0,0 +1,58 @@
+---
+pageType: howto
+title: RBAC projects
+description: Understand how n8n uses project for RBAC. Learn how to create and manage projects.
+---
+
+/// info | Feature availability
+RBAC is available on all plans except the Community edition. Different plans have different numbers of projects and roles:
+
+* Starter: one project, with admin role.
+* Pro: three projects, with admin and editor roles.
+* Enterprise: unlimited projects, with admin and editor roles.
+///
+
+n8n uses projects to group workflows and credentials, and assigns [roles](/user-management/rbac/role-types/) to users in each project. This means that a single user can have different roles in different projects, giving them different levels of access.
+
+## Create a project
+
+Instance owners and instance admins can create projects.
+
+To create a project:
+
+1. Select <span class="inline-image">![Plus icon](/_images/common-icons/plus.png)</span> **Add project**.
+1. Fill out the project settings.
+1. Select **Save**.
+
+## Add and remove users in a project
+
+To add a user to a project:
```

Comment:
Maybe worth noting that instance-level members that are admins in a project can perform those management actions.